### DISCUSSION TITLE
[Bug]: CLI Tools & Applications installed but not viewed by pimcore system requirements.

### DISCUSSION THREAD

**Discussion by alexmf23** [pimcore/pimcore#11791] [Q&A / Support] [answered]
### Expected behavior

I have installed ghostscript,libreoffice,pdftotext,timeout, but in System requirement check is not recognized.
<img width="731" alt="Screenshot 2022-03-25 at 12 43 42" src="https://user-images.githubusercontent.com/63473281/160115275-796af47e-9a3c-4145-8f7c-80d6b83fd659.png">

if i check with bin/console pimcore:system:requirements:check the system does see them
<img width="699" alt="Screenshot 2022-03-25 at 12 45 42" src="https://user-images.githubusercontent.com/63473281/160135520-aa87582b-bd93-4739-abca-fdb4eea6cecd.png">

### Actual behavior

Pimcore system requirements doesnt see the additional tools installed.

### Steps to reproduce

Regular pimcore install on centos with plesk panel.

**Comment by dvesh3** [comment, marked as answer]
@alexmf23 you might need to set `$PATH` env. variable or define individual paths for executables for Pimcore to identify these tools. please see https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/Additional_Tools_Installation.html#page_Additional-Tools-Installation

**Comment by dvesh3** [comment]
@alexmf23 Is it resolved? if so, then we can close the issue. thanks!

**Comment by alexmf23** [comment]
> @alexmf23 Is it resolved? if so, then we can close the issue. thanks!

Solved, thanks. We can close the issue.
